src/components/DraftbotBreakdown.js

```diff
@@ -210,45 +217,24 @@ export const Internal = ({ cardsInPack, draft, pack, picks, picked, seen }) => {
     return res;
   }, [cardsInPack]);
 
-  const { items, requestSort, sortConfig } = useSortableData(counts, { key: 'Total', direction: 'descending' });
-
   return (
     <>
       <Label check className="pl-2">
         <Input type="checkbox" onClick={toggleNormalized} /> Normalize the Columns
       </Label>
-      <Table bordered responsive className="small-table mt-lg-3">
-        <thead>
-          <tr>
-            <td />
-            {TRAITS.map((trait) => (
-              <HeaderCell
-                label={trait.name}
-                fieldName={trait.name}
-                sortConfig={sortConfig}
-                requestSort={requestSort}
-                tooltip={trait.description}
-              />
-            ))}
-          </tr>
-        </thead>
-        <tbody className="breakdown">
-          {items.map((item) => (
-            <tr key={item.card.details.cardID}>
-              <th scope="col">
-                <AutocardItem key={item.card.index} card={item.card} data-in-modal index={item.card.index}>
-                  <a href={`/tool/card/${encodeName(item.card.cardID)}`} target="_blank" rel="noopener noreferrer">
-                    {item.card.details.name}
-                  </a>
-                </AutocardItem>
-              </th>
-              {TRAITS.map((trait) => (
-                <td key={trait.name}>{item[trait.name]}</td>
-              ))}
-            </tr>
-          ))}
-        </tbody>
-      </Table>
+      <SortableTable
```

Comment:
As is, this table doesn't fit horizontally on my monitor (at fullscreen 16:9 1080p). Unless you intended to enlarge the table, you should retain the `small-table` class name.

Reply:
It would also be a nice touch-up to add some whitespace between the sections of the breakdown page so that the headings aren't so close to the elements above them.